### DISCUSSION TITLE
2023/03/05/changyongtaolu/

### DISCUSSION THREAD

**Discussion by giscus** [qwq2482/qwq2482.github.io#6] [General]
# 2023/03/05/changyongtaolu/

行为：  发布动态或者是专栏配上感人的文案，以及几张病例单和患者在病床上的照片，最后放上收款码，让别人给他打钱，但是你仔细看还是能看到漏洞的，下面会解释。  可疑点： 1.IP对不上，在全球各地飞来飞去，比如他发的病例显示在福建，但是空间IP可能又会显示在安徽（还有一次是在美国），然后到了评论又飞到了香港或台湾，为什么到全球各地飞呢？懂的人都懂           2.设置评论发布限制，真正的求助

https://qwq2482.github.io/2023/03/05/changyongtaolu/

**Comment by ghost** [comment]
This just a test  
![test](https://qwq2482.github.io/zpzzimg/default.jpg)  
How To Send image?  
use:  
````
![imgname](url)
````
to send

**Comment by ghost** [comment]
如果你有链接可以链接到图片的话，可以使用
````
![图片名称](链接)
````
来发送图片，没有就在[这里](https://github.com/qwq2482/qwq2482.github.io)发个issues
